### PR TITLE
Add backup script and travis tests

### PR DESCRIPTION
**Pull request by alachaum**

### CODE REVIEW

**Comment by alachaum** [comment]
@x4d3 would you be able to review this one?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+script:
+  # Build docker image
+  - cd 2.6/
+  - docker build -t maestrano/mongo:travis . > /dev/null 2>&1
+
+  # Create volumes
+  - mkdir -p vol-m0 vol-m1 vol-m2
+
+  # Configuration
+  - HOST_PUB_IP=$(ifconfig | grep "eth0 " -A 1 | grep "inet " | cut -d':' -f2 | cut -d' ' -f1)
```
Comment (x4d3) [inline .travis.yml]: I was wondering regarding this example.
Why not 
`ip route get 8.8.8.8 | awk '{print $NF; exit}'`
https://askubuntu.com/questions/430853/how-do-i-find-my-internal-ip-address/604691#604691

Comment (alachaum) [reply .travis.yml]: Good one :) I'll use that going forward

File: 2.6/backup.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# This script creates a full backup of the mysql database
```
Comment (x4d3) [inline 2.6/backup.sh]: typo mysql -> mongo

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,201 @@
+---
+sudo: required
+dist: trusty
+
+services:
+  - docker
+
+script:
+  # Build docker image
+  - cd 2.6/
```
Comment (x4d3) [inline .travis.yml:10]: maybe extract in a travis.sh ? In order to have code colouring and all these things.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+
+  #============================================================================
+  # Test standalone Mongo
+  #============================================================================
+  # Start standalone instance
+  - docker run -d -p $PORT_NODE_0:27017 -v $VOL_NODE_0:/data -e MONGO_USER=root -e MONGO_PASSWORD=root --name m0 maestrano/mongo:travis
+  - sleep 5
+
+  # Wait for standalone instance to be up
+  - for i in {30..0}; do mongo -u root -p root --quiet --eval 'rs.isMaster().ok' $HOST_PUB_IP:$PORT_NODE_0/admin &> /dev/null && break; sleep 1; done
```
Comment (x4d3) [inline .travis.yml:36]: maybe extract the wait commands in a function

File: .travis.yml (hunk 3 above)
Comment (alachaum) [reply .travis.yml:10]: I'm still unsure about this - the benefit of having commands inside the travis.yml is to see the return codes step by step. We could maybe achieve this with a "set -e". I may keep it like that for the moment and refactor later

File: .travis.yml (hunk 4 above)
Comment (alachaum) [reply .travis.yml:36]: I will if we eventually move to a travis.sh script. For now I prefer to keep it simple.

**Review by x4d3 — APPROVED** [review]
I approve

**Pull request merged**